deploy/local_single_user_deploy.sh

```diff
@@ -2,37 +2,68 @@
 
 # Variables ____________________________________________________________________________________________________________
 # Define some colors
-ERR='\033[1;31m\033[41m'
-INFO='\033[1,36m\033[42m'
+ERR='\033[7;37m\033[41m'
+INFO='\033[7;49m\033[92m'
 DEBUG='\033[0;33m'
 NC='\033[0m' # No Color
+
 NUM_KEEPERS=1
+NUM_GRAPHERS=1
 
 # Directories
 WORK_DIR="/home/${USER}/chronolog"
```

Comment:
Quick question on this. Work Dir is configurable, but it has a default value in case the parameter is not added. You are requesting to delete the default value and force the user to add this parameter when running the script?

Directory existence is checked. Permissions need to be checked. I will add this.

Reply:
yes, I prefer no default in this case. The user is required to provide the directory path to where the deployment would be staged.